pkg/services/sqlstore/migrations/ualert/alert_rule.go

```diff
@@ -100,7 +100,6 @@ func addMigrationInfo(da *dashAlert) (map[string]string, map[string]string) {
 func (m *migration) makeAlertRule(cond condition, da dashAlert, folderUID string) (*alertRule, error) {
 	lbls, annotations := addMigrationInfo(&da)
 	name := normalizeRuleName(da.Name)
-	lbls["alertname"] = name
```

Comment:
What's the reason for deleting this?

Reply:
Nvm, got my answer from [this PR](https://github.com/grafana/grafana/pull/56509)

Reply:
artifact from other PR. rebased.